helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -246,6 +250,10 @@ public String getHostname() {
     return hostname;
   }
 
+  public Date getCreated() {
+    return created;
+  }
```

Comment:
`Date` is itself not immutable (you can call e.g. `setHour()` on `Date`), so even if the reference remains unchanged this effectively makes `Job` mutable.